gen3/neural-networks/advanced-examples/ocr/ocr/README.md

```diff
@@ -1,31 +1,39 @@
-## Text Detection + Optical Character Recognition (OCR) Pipeline
+# Overview
+We provide here an example for running a two stage text detection and OCR pipeline. This example uses PaddlePaddle text detection and text recognition (OCR) models from [HubAI Model ZOO](https://hub.luxonis.com/ai/models). The example visualizes the recognized text on an adjacent white image in the locations in which it was detected. This example showcases how a twostage pipeline can easily be built using depthai.
 
-This pipeline implements text detection (EAST) followed by optical character recognition of the detected text.
 
-## Installation
+**WARNING:** As of depthai alpha10 the example only works on OAK4 devices. 
 
-```
-python3 -m pip install -r requirements.txt
-```
-
-## Usage
 
-Run the application
-
-```
-python3 main.py
+# Instalation
+Running this example requires a **Luxonis OAK4 device** connected to your computer. You can find more information about the supported devices and the set up instructions in our [Documentation](https://rvc4.docs.luxonis.com/hardware).
+Moreover, you need to prepare a **Python 3.10** environment with [DepthAI](https://pypi.org/project/depthai/) and [DepthAI Nodes](https://pypi.org/project/depthai-nodes/) packages installed. You can do this by running:
```

Comment:
Where does py3.10 dependency come from since `depthai` and `depthai-nodes` should both work also with 3.8?